telemetry/specs/INT.mdk

```diff
@@ -1009,24 +1006,28 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   Details of the metadata semantics YANG model can be accessed at the link below:
   https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-dtel-metadata-semantics.yang
   
-  Bits 0 - 13 are Baseline INT Instructions and Bit 14, Domain Specific Instruction.
-  Domain Specific Instruction is an instruction that requires additional 
-  processing of Domain Specific ID, Domain Specific Flags (DS Flags) and 
-  Domain Specific Field (DS Field). If the Domain Specific ID doesn't match the 
-  Domain ID of the node, then the transit node is required to pad the node's 
-  INT Metadata stack with the special all-ones reserved value for Domain Specific 
+  Bits 0 - 14 are Baseline INT Instructions.
+  
+  * Domain Specific ID (16b): the unique ID of the INT Domain.
+  
+  * DS Instruction (16b): Instruction bit map specific to the INT domain identified by the 
+  Domain Specific ID. Domain Specific Instruction is an instruction that requires additional 
+  processing of Domain Specific Flags (DS Flags) for the INT Domain identified by Domain Specific ID. 
+  If the Domain Specific ID doesn't match the Domain ID of the node, then the transit node is 
+  required to pad the node's INT Metadata stack with the special all-ones reserved value for Domain Specific 
   Metadata length as calculated by subtracting the Baseline Metadata Length from Hop_ML.
   
   If the Domain Specific ID matches the node's Domain ID then additional processing 
-  of the Domain Specific Flags and Domain Specific Field is required and 
+  of the Domain Specific Flags and Domain Specific Instruction is required and 
   Domain Specific Metadata is appended to the Baseline Metadata before 
-  Checksum Complement is inserted.
+  Checksum Complement is inserted. The amount of Domain Specific Metadata
+  must be a multiple of 4 bytes, determined from the Domain Specific
+  Instruction and consistent with the per-hop metadata length (Hop ML) set by the
+  INT source.
 
   Each instruction requests 4 bytes of metadata to be inserted at each hop,
-  except if bit 6 and bit 14 is set. If bit 6 is set, the instruction requires 
-  8 bytes of metadata. If bit 14 is set, the instruction requires a domain specific
-  metadata of n bytes, n being a multiple of 4 bytes. Per-hop metadata length is 
-  set accordingly at the INT source.
+  except if bit 6. If bit 6 is set, the instruction requires 
+  8 bytes of metadata. Per-hop metadata length is set accordingly at the INT source.
```

Comment:
This paragraph should be moved up to the Instruction bitmap section, in front of the Domain Specific fields.

Reply:
Makes sense. Done.